packages/vesting-base/src/testing.rs

```diff
@@ -14,14 +14,21 @@ fn proper_initialization() {
 
     let msg = InstantiateMsg {
         owner: "owner".to_string(),
-        vesting_token: token_asset_info(Addr::unchecked("ntrn_token")),
+        token_info_manager: "token_info_manager".to_string(),
         vesting_managers: vec!["manager1".to_string(), "manager2".to_string()],
     };
 
     let env = mock_env();
     let info = mock_info("addr0000", &[]);
+    let _res = vest_app.instantiate(deps.as_mut(), env, info, msg).unwrap();
+
+    let msg = ExecuteMsg::SetVestingToken {
+        vesting_token: token_asset_info(Addr::unchecked("ntrn_token")),
```

Comment:
I'd like to see more tests here:
1. no vesting token — msgs don't work
2. if you like the suggestion about the set msg executable only once — this kind of tests
3. update manager

the other contracts changes are very similar, so please consider similar tests to add to all of them

Reply:
well, this particular test only tested instantiation, so I don't go to full cover the contract in unit test, almost everything is covered by integration tests